strr-examiner-web/app/layouts/examiner.vue

```diff
@@ -1,37 +1,50 @@
 <script setup lang="ts">
+import { RoutesE } from '~/enums/routes'
 import { useExaminerStore } from '~/store/examiner'
 
 const { isAuthenticated } = useKeycloak()
 const headerOptions = useAppConfig().connect.core.header.options
 provide(headerOptionsSymbol, headerOptions)
 const localePath = useLocalePath()
+const { loading } = storeToRefs(useConnectDetailsHeaderStore())
+const route = useRoute()
 
 const { getNextApplication } = useExaminerStore()
 
 const items = [{
   key: 'dashboard',
   label: 'Dashboard',
   icon: 'i-mdi-list-box-outline',
-  path: '/dashboard'
+  path: RoutesE.DASHBOARD
 }, {
   key: 'examine',
   label: 'Examine',
   icon: 'i-mdi-file-search-outline',
-  path: '/dashboard'
+  path: RoutesE.EXAMINE
 }]
 
+const activeTab = computed(() => route.path.includes(RoutesE.EXAMINE) ? 1 : 0)
+
 async function onChange (index: number) {
+  loading.value = true
   const item = items[index]
 
-  const nextApp = await getNextApplication()
-  const navigateToPath = index === 0 ? item.path : `${item?.path}/${nextApp}`
+  let navigateToPath = item.path as string
+
+  if (item.path.includes(RoutesE.EXAMINE)) {
+    const nextApp = await getNextApplication()
+    navigateToPath = `${item?.path}/${nextApp}`
+  }
+
   return navigateTo(localePath(navigateToPath))
 }
 
 </script>
 <template>
   <div>
-    <ConnectHeaderWrapper>
+    <ConnectHeaderWrapper
```

Comment:
If you need a change to this at all I can get a new version out relatively quickly.

Reply:
You mean if I need to adjust anything in this component? Yea, sounds good. So far ok, just some style updates, but if anything, I let you know, thanks!

Reply:
If the padding or whatever is causing an issue.